util/build.sh

```diff
@@ -11,8 +11,8 @@ force=$2
 
 ngx-build $force $version \
             --with-http_ssl_module \
-            --with-cc-opt="-I$OPENSSL_INC -I$PCRE_INC" \
-            --with-ld-opt="-L$OPENSSL_LIB -L$PCRE_LIB -Wl,-rpath,$OPENSSL_LIB:$PCRE_LIB:$LIBDRIZZLE_LIB" \
+            --with-cc-opt="-I$OPENSSL_INC" \
+            --with-ld-opt="-L$OPENSSL_LIB -Wl,-rpath,$OPENSSL_LIB" \
```

Comment:
Why this change? It looks like an accident check in.

Reply:
@agentzh 
This build script will fail if we provide `OPENSSL_LIB` but not `$PCRE_LIB` and `$LIBDRIZZLE_LIB`. Since we only care about OpenSSL, I modify this script to let the build pass.

Reply:
@spacewander I think we should always support specifying our own PCRE. We don't want to use the system PCRE anyway. Better revert this.

Reply:
@agentzh 
PCRE and libdrizzle are added in the travis build via a new commit now.